lib/takeaway.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'dish.rb'
+require_relative 'text.rb'
+require 'twilio-ruby'
+
+class Takeaway 
+  attr_reader :menu, :final_order, :texter
+  def initialize
+    @texter = Text.new
+    @menu = {
+      "dish 1" => 5,
+      "dish 2" => 7,
+      "dish 3" => 9}
+    @final_order = []
+  end
+
+  def menu
+    @menu.each do |dish, price|
+      puts "#{dish} -- £#{price}"
+    end
+  end
+
+  def add_dish(name, price)
```

Comment:
Method definitely would work in production but dependency injection might be a better way to implement the creating of a class instance when using rspec tests?